index.js

```diff
@@ -181,8 +233,8 @@ medias = (async function () {
     return clampDuration(duration)
   }
 
-  const videoElements = getAllVideoElements() ?? []
-  const audioElements = getAllAudioElements() ?? []
+  const videoElements = document.querySelectorAll('video') ?? []
```

Comment:
I don't think this can ever return `null` or `undefined`

```suggestion
  const videoElements = document.querySelectorAll('video')
```

Reply:
Nice!